source/_config.yml

```diff
@@ -21,13 +21,28 @@ parse:
     - substitution
     - tasklist
   myst_substitutions:
+    # Authors
     田冬冬: "[田冬冬](https://me.seisman.info/)"
     姚家园: "[姚家园](https://github.com/core-man)"
     王亮: "[王亮](https://github.com/wangliang1989)"
     赵志远: "[赵志远](https://github.com/zhaozhiyuan1989)"
+    # Platforms
     Linux: "{fab}`linux;fa-brands`"
     macOS: "{fab}`apple;fa-brands`"
     Windows: "{fab}`windows;fa-brands`"
+    # Common notes
+    install_notes: |
+      :::{dropdown} 阅读须知
+      1.  我们假定用户已经根据《{doc}`seis101:computer/setup`》中的内容安装了
+          C/C++ 和 Fortran 编译器 `gcc`、`g++` 和 `gfortran`
+          以及常用命令行工具 `make` 和 `git` 等。
+      2.  修改环境变量时，我们假定用户当前使用的 Shell 是 Bash，且 Bash 配置文件
+          为 `~/.bashrc`。Z Shell （zsh）用户应修改 `~/.zshrc`。
+          不确定自己当前使用的是何种 Shell 的用户，可检查 `echo $SHELL` 命令的输出。
+      3.  所有软件的安装方式及安装路径，均遵循
+          {doc}`《文件管理实践经验》<seis101:best-practices/file-organization>` 和
+          {doc}`《软件安装实践经验》<seis101:best-practices/software-installation>`。
+      :::
```

Comment:
Define the notes in the `_config.yml` file and use it in other Markdown files using the `{{ install_notes }}` syntax.